synchronizer/synchronizer_test.go

```diff
@@ -28,14 +28,18 @@ type mocks struct {
 	EthTxManager *ethTxManagerMock
 	DbTx         *dbTxMock
 	ZKEVMClient  *zkEVMClientMock
+	//EventLog     *eventLogMock
 }
 
+//func Test_Given_StartingSynchronizer_When_CallFirstTimeExecutor_Then_StoreProverID(t *testing.T) {
```

Comment:
We should remove commented code.

Reply:
I left this because is missing the unittest and I'm going to do it after the merge of the solution